.github/workflows/test.yml

```diff
@@ -32,16 +36,98 @@ jobs:
           arch: x64
           os: windows-latest
           toxenv: py37
+        - name: ubuntu-py37
+          python: '3.7'
+          arch: x64
+          os: ubuntu-latest
+          toxenv: py
+          tox_extra_args: "-n 2"
+        - name: ubuntu-py38
+          python: '3.8'
+          arch: x64
+          os: ubuntu-latest
+          toxenv: py
+          tox_extra_args: "-n 2"
+        - name: ubuntu-py36-mypyc
+          python: '3.6'
+          arch: x64
+          os: ubuntu-latest
+          toxenv: py
+          tox_extra_args: "-n 2"
+          test_mypyc: true
+        - name: ubuntu-py39-mypyc
+          python: '3.9'
+          arch: x64
+          os: ubuntu-latest
+          toxenv: py
+          tox_extra_args: "-n 2"
+          test_mypyc: true
+        - name: macos-py36
+          python: '3.6'
+          arch: x64
+          os: macos-latest
+          toxenv: py
+          tox_extra_args: "-n 2 mypyc/test/test_run.py mypyc/test/test_external.py"
+        - name: ubuntu-py-36-debug-build
+          python: '3.6.8'
+          arch: x64
+          os: ubuntu-latest
+          toxenv: py36
+          tox_extra_args: "-n 2 mypyc/test/test_run.py mypyc/test/test_external.py"
+          debug_build: true
+        - name: type-checking
+          python: '3.7'
+          arch: x64
+          os: ubuntu-latest
+          toxenv: type
+        - name: code-style
+          python: '3.7'
+          arch: x64
+          os: ubuntu-latest
+          toxenv: lint
 
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
         architecture: ${{ matrix.arch }}
-    - name: install tox
+    - name: Debug build
+      if: ${{ matrix.debug_build }}
+      run: |
+        PYTHONVERSION=${{ matrix.python }}
+        PYTHONDIR=~/python-debug/python-$PYTHONVERSION
+        VENV=$PYTHONDIR/env
+        ./misc/build-debug-python.sh $PYTHONVERSION $PYTHONDIR $VENV
+        source $VENV/bin/activate
+    - name: Install tox
       run: pip install --upgrade 'setuptools!=50' 'virtualenv<16.7.11' tox==3.20.1
-    - name: setup tox environment
+    - name: Compiled with Mypyc
```

Comment:
s/Mypyc/mypyc/

The "mypy style guide" is to only capitalize mypy at the start of sentences. (I hate doing that, so I try not to put it at the start of sentences :P)

(I saw that some of facebook's python performance work referred to us as "MyPyC" which hurt a little)

Reply:
done